docs/configuring-playbook-dimension.md

```diff
@@ -0,0 +1,48 @@
+# Setting up Dimension (optional)
+
+Dimension integrations manager installation is disabled by default. You can enable it in your configuration file (`inventory/matrix.<your-domain>/vars.yml`):
+
+```yaml
+matrix_dimension_enabled: true
+```
+Your Matrix also needs federation enabled for Dimension to work, so if you have disabled it by setting `matrix_synapse_federation_enabled` to `false`, please remove or comment this variable in your configuration file. Or set it to:
```

Comment:
Since federation being enabled (`matrix_synapse_federation_enabled: true`) is the default option and people need to go out of their way to disable it, I wonder if we should pay that much attention to it here.

For most people, it does water down the message of this documentation entry with needless information.

Because we have a nice `validate_config.yml`, those that enable it will be made aware that federation is necessary.
It might still make sense to have a tooltip on this page though, to prevent someone wasting some effort on Dimension, only to find out that they can't use it do to federation being disabled. So we might mention it, but maybe with less text/codeblocks. It might also be a good idea to say that federation being enabled is the default, so that we don't throw in most people into wondering about what their settings are.

Example:

For an Integration Manager like Dimension to work, your server needs to have federation enabled (`matrix_synapse_federation_enabled: true`). This is the default for this playbook, so unless you've explicitly disabled federation, you're okay.